src/notifications/NotificationsHandler.tsx

```diff
@@ -300,6 +301,19 @@ export const NotificationsHandler = ({ walletReady }: Props) => {
         `NotificationsHandler: handling wallet connect notification`,
         { notification }
       );
+    } else if (type === NotificationTypes.marketing) {
+      logger.info(`NotificationsHandler: handling marketing notification`, {
+        notification,
+      });
+      const data = (notification.data as unknown) as MarketingNotificationData;
```

Comment:
idk what double type casting does but just noting it in case its not necessary

Reply:
can't cast it to `MarketingNotificationData` without casting to unknown first, TS bs